solidity/ecdsa/test/utils/dkg.ts

```diff
@@ -56,6 +56,7 @@ export async function signAndSubmitCorrectDkgResult(
   signers: Operator[]
   dkgResult: DkgResult
   dkgResultHash: string
+  members: number[]
```

Comment:
`dkgResult` already has `members` field and we can use that in tests.

Reply:
Yeah, very good point. I reverted this change in https://github.com/keep-network/keep-core/pull/2868/commits/8d88718b92b1c98cbe26feccfc2cd8a107138e42.